core-blocks/audio/edit.js

```diff
@@ -53,6 +60,14 @@ class AudioEdit extends Component {
 			this.setState( { editing: false } );
 		};
 
+		const onToggleAutoplay = ( newVal ) => {
+			setAttributes( { autoplay: newVal } );
+		};
+
+		const onToggleLoop = ( newVal ) => {
```

Comment:
Seems like it'd be nicer to have:

```js
toggleAttribute = ( attribute ) => ( newValue ) => {
  setAttributes( { [ attribute ]: newValue } );
}
```

and then have `onChange={ toggleAttribute( 'loop' ) }`

Reply:
And also: why is this function set as a constant inside every `render()` call? This should be a component method.

Reply:
Refactored in 275ef72d3b349410d62d4c4880330cb977607932